public/wp-content/themes/glific/dist/images/blog-delaut-image.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please optimize and minify. Also, please self-review once.

Reply:
👍